test/e2e/kubectl/kubectl.go

```diff
@@ -1046,6 +1048,46 @@ metadata:
 		})
 	})
 
+	framework.KubeDescribe("Kubectl copy", func() {
+		podYaml := substituteImageName(string(readTestFileOrDie("busybox-pod.yaml.in")))
```

Comment:
just name this file .yaml if it is a valid yaml file that can be created directly